### PR TITLE
more powerfull logging for Neutron

### PR DESCRIPTION
**Pull request by xenolog**

### CODE REVIEW

**Comment by xenolog** [comment]
https://blueprints.launchpad.net/fuel/+spec/neutron-server-stderr-logging

File: deployment/puppet/neutron/files/ocf/neutron-agent-dhcp [hunk 1]
```diff
@@ -129,6 +131,14 @@ The pid file to use for this OpenStack DHCP Service (neutron-dhcp-agent) instanc
 <content type="string" default="${OCF_RESKEY_pid_default}" />
 </parameter>
 
+<parameter name="log_file" unique="0" required="0">
```
Comment (bogdando) [inline deployment/puppet/neutron/files/ocf/neutron-agent-dhcp:138]: What is it? According to blueprint, neutron-server configuration file and init script should not use log_file

Comment (xenolog) [inline deployment/puppet/neutron/files/ocf/neutron-agent-dhcp:138]: Neutron-agents must log trought stderr too. This parameter specify filename for redirecting stderr inside OCF script.

**Comment by xenolog** [comment]
Do not merge, need more deep testing

File: deployment/puppet/neutron/manifests/agents/dhcp.pp [hunk 2]
```diff
@@ -59,6 +59,11 @@
   neutron_dhcp_agent_config {
     'DEFAULT/debug':             value => $debug;
     'DEFAULT/verbose':           value => $verbose;
+    'DEFAULT/log_dir':          ensure => absent;
+    'DEFAULT/log_file':         ensure => absent;
+    'DEFAULT/log_config':       ensure => absent;
+    'DEFAULT/use_syslog':       ensure => absent;
+    'DEFAULT/use_stderr':       ensure => absent;
```
Comment (bogdando) [inline deployment/puppet/neutron/manifests/agents/dhcp.pp:66]: it should be TRUE

File: deployment/puppet/neutron/manifests/agents/l3.pp [hunk 3]
```diff
@@ -47,6 +47,11 @@
   neutron_l3_agent_config {
     'DEFAULT/debug':          value => $debug;
     'DEFAULT/verbose':        value => $verbose;
+    'DEFAULT/log_dir':       ensure => absent;
+    'DEFAULT/log_file':      ensure => absent;
+    'DEFAULT/log_config':    ensure => absent;
+    'DEFAULT/use_syslog':    ensure => absent;
+    'DEFAULT/use_stderr':    ensure => absent;
```
Comment (bogdando) [inline deployment/puppet/neutron/manifests/agents/l3.pp:54]: it should be TRUE

File: deployment/puppet/neutron/manifests/agents/metadata.pp [hunk 4]
```diff
@@ -22,6 +22,12 @@
 
   neutron_metadata_agent_config {
     'DEFAULT/debug':              value => $debug;
+    'DEFAULT/verbose':            value => $verbose;
+    'DEFAULT/log_dir':           ensure => absent;
+    'DEFAULT/log_file':          ensure => absent;
+    'DEFAULT/log_config':        ensure => absent;
+    'DEFAULT/use_syslog':        ensure => absent;
+    'DEFAULT/use_stderr':        ensure => absent;
```
Comment (bogdando) [inline deployment/puppet/neutron/manifests/agents/metadata.pp:30]: it should be TRUE

File: deployment/puppet/neutron/manifests/plugins/ovs.pp [hunk 5]
```diff
@@ -67,6 +67,11 @@
     'AGENT/polling_interval':       value => $neutron_config['polling_interval'];
     'AGENT/root_helper':            value => $neutron_config['root_helper'];
     'SECURITYGROUP/firewall_driver': value => 'neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver';
+    'DEFAULT/log_dir':        ensure => absent;
+    'DEFAULT/log_file':       ensure => absent;
+    'DEFAULT/log_config':     ensure => absent;
+    'DEFAULT/use_syslog':     ensure => absent;
+    'DEFAULT/use_stderr':     ensure => absent;
```
Comment (bogdando) [inline deployment/puppet/neutron/manifests/plugins/ovs.pp:74]: it should be TRUE

**Pull request closed without merging**